### ISSUE TITLE
Colab not Working !!! Error when importing `datapipeline`

### ISSUE THREAD

**Issue by SauravMaheshkar** [aqlaboratory/openfold#13]
Great work with reproducing the original code and creating a OpenSource PyTorch Implementation ☕️☕️☕️☕️

---

When I try to run the attached Colab Notebook, In the "Search against genetic databases" subsection while importing `datapipeline` from `openfold.data`, I run into a ImportError, viz.

```bash 
ImportError: cannot import name 'MultipleChainsError' from 'openfold.data.templates' (/opt/conda/lib/python3.7/site-packages/openfold/data/templates.py)
```

The full traceback is attached below :-

<details>

```bash

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-7-8051d602620b> in <module>()
     27 from openfold.data import feature_pipeline
     28 from openfold.data import parsers
---> 29 from openfold.data import data_pipeline
     30 from openfold.data.tools import jackhmmer
     31 from openfold.model import model

/opt/conda/lib/python3.7/site-packages/openfold/data/data_pipeline.py in <module>()
     20 import numpy as np
     21 
---> 22 from openfold.data import templates, parsers, mmcif_parsing
     23 from openfold.data.tools import jackhmmer, hhblits, hhsearch
     24 from openfold.data.tools.utils import to_date

/opt/conda/lib/python3.7/site-packages/openfold/data/templates.py in <module>()
     26 import numpy as np
     27 
---> 28 from openfold.data import parsers, mmcif_parsing
     29 from openfold.data.tools import kalign
     30 from openfold.data.tools.utils import to_date

/opt/conda/lib/python3.7/site-packages/openfold/data/mmcif_parsing.py in <module>()
     27 import numpy as np
     28 
---> 29 from openfold.data.templates import MultipleChainsError
     30 import openfold.np.residue_constants as residue_constants
     31 

ImportError: cannot import name 'MultipleChainsError' from 'openfold.data.templates' (/opt/conda/lib/python3.7/site-packages/openfold/data/templates.py)

```

</details>

Interesting enough if I add `from openfold.data.templates import MultipleChainsError` I run into a circular `ImportError`, the error trace is attached below

<details>

```bash
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-8-71256580fa0c> in <module>()
     27 from openfold.data import feature_pipeline
     28 from openfold.data import parsers
---> 29 from openfold.data.templates import MultipleChainsError
     30 from openfold.data import data_pipeline
     31 from openfold.data.tools import jackhmmer

/opt/conda/lib/python3.7/site-packages/openfold/data/templates.py in <module>()
     26 import numpy as np
     27 
---> 28 from openfold.data import parsers, mmcif_parsing
     29 from openfold.data.tools import kalign
     30 from openfold.data.tools.utils import to_date

/opt/conda/lib/python3.7/site-packages/openfold/data/mmcif_parsing.py in <module>()
     27 import numpy as np
     28 
---> 29 from openfold.data.templates import MultipleChainsError
     30 import openfold.np.residue_constants as residue_constants
     31 

ImportError: cannot import name 'MultipleChainsError' from 'openfold.data.templates' (/opt/conda/lib/python3.7/site-packages/openfold/data/templates.py)

```

</details>

**Comment by gahdritz** [comment]
Sorry about that! One of the bugfixes yesterday introduced a circular import. Fixed in 0567d99

**Comment by tcapelle** [comment]
that was fast, I was coming to post the same thing.
